test/test_configs/config.fixtures_dynamodb.test.yml

```diff
@@ -1,6 +1,7 @@
-fixture_loader_enabled: true
-
 aws_cloudwatch_endpoint: http://172.17.0.1:4582
 aws_dynamoDb_endpoint: http://172.17.0.1:4569
 aws_dynamoDb_autoCreate: true
 aws_sdk_retries: 30
+
+fixtures:
```

Comment:
should we name this fixture_loader instead?

Reply:
Will keep it as it is to be consistent and use the package name.